AmericanMedicalAssociation/ama-style-guide-2
# EWL-11016: Update anchor link functionality and positioning.

# Pre-PR Checklist
If you have access to approved developer tools such as AMA co-pilot, ask it the following and test any suggestions before posting the PR.


## JIRA Ticket(s)
- [EWL-11016: Page Grouper | Anchor link targets need to be repositioned](https://ama-it.atlassian.net/browse/EWL-11016)


## Description:
Adjust script and styling so anchor links within page grouper (and anywhere else) dont get cut off by any header/sticky elements.


## To Test:
- link ama-d8 pr locally: https://github.com/AmericanMedicalAssociation/ama-d8/pull/5049
- clear caches
- add/edit a page grouper block to have anchor links
- on article with page grouper make sure table of content is enabled
- confirm any anchor link clicks (both through page grouper and TOC) scroll to selected entry and do not get cut off by any header elements
- confirm for desktop and mobile


## Visual Regressions
Ensure any available regression testing has been run.


## Relevant Screenshots/GIFs
Use something like [Skitch](https://evernote.com/skitch/) or [GIPHY Capture](https://giphy.com/apps/giphycapture) to capture images/gifs to demonstrate behaviors.


## Remaining Tasks



## Additional Notes
Anything more to add?

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
